httpbin/core.py

```diff
@@ -649,6 +648,72 @@ def links(n):
     return redirect(url_for('link_page', n=n, offset=0))
 
 
+@app.route('/file/<path:p>', methods=('GET', 'POST',))
+def file(p):
+    """ File operations - create a new file, get a file or delete a file
+    :param p: directory path of the file for POST, full file path for GET and DELETE
+    :return:
+        GET: Returns a multi-part response with requested file(s)
+        POST: Returns the full path of the created file on success, error on error
+    """
+    if not args.file_endpoint:
+        return status_code(501) # Not Implemented
+
+    root_dir = os.path.join(app.config['SPOOL_DIR'], 'files')
+    if root_dir in ("", "/") or root_dir.startswith("/root"):
+        return status_code(500)
+
+    p = os.path.join(root_dir, p)
+    filename = request.args.get("name", None)
+
+    # for security
+    if '..' in p or (filename is not None and '..' in filename):
+        return status_code(404)
+
+    if request.method == 'GET':
+        created_since = request.args.get("created_since")
```

Comment:
Only look for files created since `created_since` seconds in the past.  Change this to store the timestamp to represent `ctime`